web/app/view/DevicesController.js

```diff
@@ -74,28 +74,30 @@ Ext.define('Traccar.view.DevicesController', {
     },
 
     onCommandClick: function () {
```

Comment:
Please add some spacing to this method. It became too big and it's really hard to understand what it does.